keps/NNNN-kep-template/README.md

```diff
@@ -303,6 +303,13 @@ extending the production code to implement this enhancement.
 
 ##### Integration tests
 
+<!--
+Integration tests are contained in k8s.io/kubernetes/test/integration.
+Integration tests allow control of the configuration parameters used to start the binaries under test.
+This is different from e2e tests which do not allow configuration of parameters.
+Doing this allows testing non-default options and multiple different and potentially conflicting command line options.
+-->
```

Comment:
maybe include examples which components must have these tests and what kind of feature changes are OK not to have it?